### PR TITLE
Improve Chinese Simplified translate

### PR DESCRIPTION
**Pull request by mailgyc**
Improve Chinese Simplified translate: “仪表盘” to “控制台”, and "郵遞" in Chinese Simplified should be “邮递”

### CODE REVIEW

**Comment by solarissmoke** [comment]
Would you be able to update your PR to resolve conflicts?

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,4 @@ settings_local.py
 /sandbox/public/
 /sandbox/whoosh_index/
 /sandbox/logs/
+/oscar
```
Comment (solarissmoke) [inline .gitignore]: Why are you adding this to .gitignore?

Comment (mailgyc) [reply .gitignore]: oh, sorry, it's a mistake, it's my virtualenv folder name, you can just delete it.

**Comment by solarissmoke** [comment]
Turns out this needs to happen in transifex (https://www.transifex.com/codeinthehole/django-oscar/). Would you be able to update the translations there? They will get pulled in at the next release.

**Pull request closed without merging**